docs/tutorials/setup-development-machine-with-vagrant.md

```diff
@@ -0,0 +1,266 @@
+  --output=format:gob \
+  --output=option:parse-arguments \
+  | ./dist/tracee-rules \
+  --input-tracee=file:stdin \
+  --input-tracee=format:gob
+Loaded 14 signature(s): [TRC-1 TRC-13 TRC-2 TRC-14 TRC-3 TRC-11 TRC-9 TRC-4 TRC-5 TRC-12 TRC-8 TRC-6 TRC-10 TRC-7]
+
+*** Detection ***
+Time: 2022-03-26T18:48:00Z
+Signature ID: TRC-2
+Signature: Anti-Debugging
+Data: map[]
+Command: strace
+Hostname: ubuntu-impish
+```
+
+## Switch Between CO-RE and non CO-RE Linux Distribution
+
+By default, the development machine is running Ubuntu Linux 21.10 Impish Indri. You can see that it has a BTF-enabled
+kernel by checking the existence of the `/sys/kernel/btf/vmlinux` file.
+
+```ruby
+Vagrant.configure("2") do |config|
+  # config.vm.box = "ubuntu/focal64"     # Ubuntu 20.04 Focal Fossa (non CO-RE)
+  # config.vm.box = "ubuntu/hirsute64"   # Ubuntu 21.04 Hirsute Hippo (CO-RE)
+  config.vm.box = "ubuntu/impish64"      # Ubuntu 21.10 Impish Indri (CO-RE)
+end
+```
+
+Sometimes you may want to test Tracee with a non CO-RE distribution. You can do that by editing the Vagrantfile and
+modifying the `config.vm.box` property. For example, you can switch to Ubuntu Linux 20.04 Focal Fossa as follows:
+
+```ruby
+Vagrant.configure("2") do |config|
+  config.vm.box = "ubuntu/focal64"       # Ubuntu 20.04 Focal Fossa (non CO-RE)
+  # config.vm.box = "ubuntu/hirsute64"   # Ubuntu 21.04 Hirsute Hippo (CO-RE)
+  # config.vm.box = "ubuntu/impish64"    # Ubuntu 21.10 Impish Indri (CO-RE)
+end
+```
+
+This change requires reprovisioning the development machine:
+
+```
+vagrant destroy
+vagrant up
+```
+
+## Deploy Tracee on Kubernetes
+
+The development machine described by Vagrantfile preinstalls [MicroK8s] Kubernetes cluster, which is suitable for testing
+Tracee.
+
+```
+vagrant@ubuntu-impish:/vagrant$ microk8s status
+microk8s is running
+high-availability: no
+  datastore master nodes: 127.0.0.1:19001
+  datastore standby nodes: none
+```
+
+There's also the [kubectl] command installed to communicate with the cluster:
+
+```
+vagrant@ubuntu-impish:/vagrant$ kubectl get nodes -o wide
+NAME            STATUS   ROLES    AGE    VERSION                    INTERNAL-IP   EXTERNAL-IP   OS-IMAGE       KERNEL-VERSION      CONTAINER-RUNTIME
+ubuntu-impish   Ready    <none>   139m   v1.23.4-2+98fc2022f3ad3e   10.0.2.15     <none>        Ubuntu 21.10   5.13.0-35-generic   containerd://1.5.9
+```
+
+```
+vagrant@ubuntu-impish:/vagrant$ kubectl create ns tracee-system
+vagrant@ubuntu-impish:/vagrant$ kubectl apply -f deploy/kubernetes/tracee-postee/tracee.yaml -n tracee-system
+daemonset.apps/tracee created
+```
+
+```
+vagrant@ubuntu-impish:/vagrant$ kubectl logs -f daemonset/tracee -n tracee-system
+INFO: probing tracee-ebpf capabilities...
+INFO: starting tracee-ebpf...
+INFO: starting tracee-rules...
+Loaded 14 signature(s): [TRC-1 TRC-13 TRC-2 TRC-14 TRC-3 TRC-11 TRC-9 TRC-4 TRC-5 TRC-12 TRC-8 TRC-6 TRC-10 TRC-7]
+Serving metrics endpoint at :3366
```

Comment:
![image](https://user-images.githubusercontent.com/7395852/160454219-a52b88f7-4efd-434f-a0f3-4f01aa80666b.png)

Reply:
It's known issue #1577